src/main/java/seedu/address/logic/commands/property/FindCommand.java

```diff
@@ -1,38 +1,69 @@
 package seedu.address.logic.commands.property;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
 
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.model.Model;
 import seedu.address.model.property.AddressContainsKeywordsPredicate;
+import seedu.address.model.property.LandlordNameContainsKeywordsPredicate;
 
 /**
  * Finds and lists all properties in property list whose name contains any of the argument keywords.
  * Keyword matching is case-insensitive.
  */
 public class FindCommand extends Command {
+
     public static final String COMMAND_WORD = "find";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all properties whose names or addresses contain"
+            + " any of the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "You should only find by name or address but not both.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Example: " + COMMAND_WORD + " "
```

Comment:
Since the Find Command can only take either prefix by address or prefix by name. For the e.g. string, just show either case, such as perhaps the address case